src/language_server_protocol.rs

```diff
@@ -171,7 +171,7 @@ impl LanguageClient {
         let selectionUI = if let Some(s) = selectionUI {
             SelectionUI::from_str(&s)?
         } else if self.vim()?.eval::<_, i64>("get(g:, 'loaded_fzf')")? == 1 {
```

Comment:
Should this check for `loaded_fzf` be there? If you are using clap or any other selection UI it doesn't really matter if you have fzf is loaded or not.